docs/messaging/messaging-journey.md

```diff
@@ -0,0 +1,12 @@
+
+---
+id: messaging-journey
+title: Messaging Journey
+slug: /messaging/messaging-journey
+---
+
```

Comment:
Will be good to add one liner to give context of documentation coming something like 'In Firefox Desktop from inception to launch, a message goes through below three steps.'